README.md

```diff
@@ -34,9 +37,9 @@ resources](https://metasploit.github.io), or the [wiki].
 Contributing
 --
 See the [Dev Environment Setup][wiki-devenv] guide on GitHub, which will
-walk you through the whole process from installing all the
+help you through the whole process from installing all the
 dependencies, to cloning the repository, and finally to submitting a
-pull request. For slightly more information, see
+pull request. For slightly more information,You can see
```

Comment:
```suggestion
pull request. For slightly more information, you can see
```

Reply:
Hlo metasploit team please help how can i make a little contribution to Metasploit Framework.
my email: anshikthind@gmail.com